modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -20,11 +20,7 @@ def self.date_from_string(string)
     attr_encrypted(:form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
     attr_encrypted(:auth_headers, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller)
 
-    validate(
-      :validate_hearing_type_selection,
-      :validate_address_unless_homeless,
-      if: proc { |a| a.form_data.present? }
-    )
+    validate(:validate_hearing_type_selection, if: proc { |a| a.form_data.present? })
```

Comment:
Determined the complexity of this validation is best handled via the model. The logic required the use of the "not" keyword in the schema and could not be written in a way that would return errors tailored for the dependent field.

Reply:
Now that we're using a single validation, perhaps extracting this proc into a method for readability makes sense?

`validate :hearing_type_selection, if: pii_still_present?`

Reply:
I did that in my original and it was suggested I return to this, since I suppose the proc itself is pretty readable, but I'm with you in the "even more readable" camp. I'm going to change it. It will be more readable, especially if we refer to the pii in the method name. Done. Doing it.